contrib/grinder/scripts/tests.py

```diff
@@ -138,16 +148,20 @@ def test_init_process(self):
     single_plot_queries_agent0 = int(math.ceil(query.default_config['singleplot_per_interval']/num_query_nodes))
     multi_plot_queries_agent0 = int(math.ceil(query.default_config['multiplot_per_interval']/num_query_nodes))
     search_queries_agent0 = int(math.ceil(query.default_config['search_queries_per_interval']/num_query_nodes))
+    enum_search_queries_agent0 = int(math.ceil(query.default_config['enum_search_queries_per_interval']/num_query_nodes))
     annotation_queries_agent0 = int(math.ceil(query.default_config['annotations_queries_per_interval']/num_query_nodes))
     
```

Comment:
@VinnyQ Can we add some tests for slices for the enums ingestion thread? See 
https://github.com/VinnyQ/blueflood/blob/8488324484900a92a53f48191cef50f18112966f/contrib/grinder/scripts/tests.py#L124